Configurations.md

```diff
@@ -2353,7 +2353,11 @@ Require a specific version of rustfmt. If you want to make sure that the
 specific version of rustfmt is used in your CI, use this option.
 
 - **Default value**: `CARGO_PKG_VERSION`
-- **Possible values**: any published version (e.g. `"0.3.8"`)
+- **Possible values**: `semver` compliant values, such as defined on [semver.org](https://semver.org/) and following the following [`semver::Version`](https://docs.rs/semver/latest/semver/struct.Version.html#total-ordering) specification.
```

Comment:
Let's drop the note about `semver::Version`. That's an internal implementation detail that I don't think we need to mention in the docs.

```suggestion
- **Possible values**: `semver` compliant values, such as defined on [semver.org](https://semver.org/).
```